app/cloud_tasks/cloud_task_publishers.py

```diff
@@ -46,27 +46,29 @@ def _create_task_with_retry(
         )
         return task
 
-    def create_task(self, body: bytes, queue_name: str, function_name: str) -> Task:
-        logger.info("creating cloud task")
+    def create_task(
+        self, body: bytes, queue_name: str, function_name: str, transactionId: str
+    ) -> Task:
 
         parent = self._client.queue_path(self._project_id, "europe-west2", queue_name)
         try:
             task = self._create_task_with_retry(body, function_name, parent)
-            logger.info("task created successfully")  # pragma: no cover
+            logger.info("task created successfully", transactionId=transactionId)
             return task
         except Exception as exc:
-            logger.exception(
-                "task creation failed",
-            )
+            logger.exception("task creation failed", transactionId=transactionId)
```

Comment:
Where the transaction id is referenced outside of a fulfilment request, I think we should consider qualifying it e.g. `fulfilment_request_transaction_id` to make a clear distinction between this and the `tx_id`.

Reply:
Makes sense 👍🏻

Reply:
Done